Cargo.toml

```diff
@@ -378,8 +377,12 @@ uucore = { version=">=0.0.9", package="uucore", path="src/uucore", features=["en
 walkdir = "2.2"
 atty = "0.2"
 
-[target.'cfg(unix)'.dev-dependencies]
+[target.'cfg(target_os = "linux")'.dependencies]
```

Comment:
I think you accidentally made this a normal dependency instead of a `dev-dependency`

Reply:
well spotted, thanks!